Governance/What-Is-a-Project.md

```diff
@@ -1,13 +1,10 @@
 
 ## Types of Projects
 
 There are two types of projects:
 
-
-
-1. **Strategy Initiated**: Every year moja global develops a [strategic plan](https://github.com/moja-global/.github/blob/master/Governance/Strategic-Plan.md). The plan contains various projects. Anybody can [propose ideas for the strategic plan](https://github.com/moja-global/.github/blob/master/Contributing/How-to-Provide-Strategic-Advice.md). The Strategy Board prioritises and approves the plan and then tries to [mobilize the necessary resources](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Assist-with-Fundraising.md) to implement the projects. Anybody can [join these projects](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md). 
-2. **User Initiated:**  As a user, you can start a project that is important to you. You can [submit a project idea](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md) anytime. The idea of the project might come from your own need / preference or it might be an issue that was already listed on the issue backlog of a repository but that needs the contribution of several persons. In either case, thank you for [starting a moja global project](https://github.com/moja-global/.github/blob/master/Contributing/How-to-Start-a-New-Project.md).  
+1. **Strategy Initiated**: Every year moja global develops a [strategic plan](https://github.com/moja-global/.github/blob/master/Governance/Strategic-Plan.md). The plan contains various projects. Anybody can [propose ideas for the strategic plan](https://github.com/moja-global/.github/blob/master/Contributing/How-to-Provide-Strategic-Advice.md). The Strategy Board prioritises and approves the plan and then tries to [mobilize the necessary resources](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Assist-with-Fundraising.md) to implement the projects. Anybody can [join these projects](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Join-moja-global.md).
+2. **User Initiated:** As a user, you can start a project that is important to you. You can [submit a project idea](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md) anytime. The idea of the project might come from your own need/preference or it might be an issue that was already listed on the issue backlog of a repository but that needs the contribution of several persons. In either case, thank you for [starting a moja global project](https://github.com/moja-global/.github/blob/master/Contributing/How-to-Start-a-New-Project.md).
```

Comment:
```suggestion
2. **User Initiated:** As a user, you can start a project that is important to you. You can [submit a project idea](https://github.com/moja-global/About-moja-global/blob/master/Contributing/How-to-Start-a-New-Project.md) anytime. The idea of the project might come from your own need/preference or it might be an issue that was already listed on the issue backlog of a repository but that needs the contribution of several people. In either case, thank you for [starting a moja global project](https://github.com/moja-global/.github/blob/master/Contributing/How-to-Start-a-New-Project.md).
```